uber/site_sections/preregistration.py

```diff
@@ -257,6 +257,7 @@ def delete(self, id):
     def dealer_confirmation(self, session, id):
         return {'group': session.group(id)}
 
+    @log_pageview
```

Comment:
In theory, if we want to be super strict about this, we could perform this ``track_pageview`` action every time we call ``session.attendee()`` from a page handler which doesn't result in an HTTPRedirect.

That's probably overkill though.